src/app/static/templates/login.ftl

```diff
@@ -19,6 +19,12 @@
                 form.classList.add('was-validated');
             }
         }
+
+        window.addEventListener('keyup', function(e) {
+            if (e.key === 'Enter') {
+                document.querySelector("#loginBtn").click();
+            }
+        });
```

Comment:
There doesn't seem to be any testing suite for this ftl page. If testing ftl is possible, would someone be able to point me to an example.

Reply:
It just captures enter presses outside of the inputs so that when you go to login and you haven't selected the inputs (perhaps because they have been auto filled by your browser), you can just press enter to click the login button.

Reply:
Oh great - that makes sense. Thanks!